client/post-editor/editor-drawer/index.jsx

```diff
@@ -241,7 +244,7 @@ class EditorDrawer extends Component {
 	}
 
 	renderSeo() {
-		const { jetpackVersionSupportsSeo } = this.props;
+		const { jetpackVersionSupportsSeo, hasConflictingSeoPlugins } = this.props;
```

Comment:
Optional, but maybe don't render either when we're missing plugin info. At the moment we can see a render then hide for a site that has a conflicting plugin.

Reply:
Thanks for catching it, updated to account for this too.